TM1py/Objects/Cube.py

```diff
@@ -112,6 +112,72 @@ def _construct_body(self) -> str:
         body_as_dict['Dimensions@odata.bind'] = [format_url("Dimensions('{}')", dimension)
                                                  for dimension
                                                  in self.dimensions]
-        if self.has_rules:
+        if str(self.rules):
```

Comment:
When a cube does not have rules str(None) returns True, this causes a rule to be created with a string "None"